src/book/02-system/03-tools/06-natnet.mdx

```diff
@@ -0,0 +1,104 @@
+---
+section: System
+chapter: Tools
+title: NatNet SDK
+description: A tool that provides positional infomation from motive mocap software.
+slug: /system/tools/natnet_sdk
+authors:
+  - Luke Haigh
+---
+
+NatNet is module that works in conjunction with the [OptiTrack Motive software](https://www.optitrack.com/software/motive/) and motion tracking camera system. The purpose of the NatNat system is to record positional information about the NUgus robots in relation to the playing field and the external envrionment so that it can be provided to our automatic anotation system which will use the positional information of the robots to automatically create segmentation masks of the environment. 
+
+## Getting NatNet Running
+
+In order to get NatNet running you will require a licenced version of the Optitrack Motive software and at least 2 [Optive Motion Capture Cameras](https://www.optitrack.com/cameras/). NatNet currently supports motive version 3.0 and below. Below is a detailed guide on how to setup motive.
```

Comment:
```suggestion
In order to run NatNet you will require a licenced version of the Optitrack Motive software and at least 2 [Optive Motion Capture Cameras](https://www.optitrack.com/cameras/). NatNet currently supports motive version 3.0 and below. Below is a detailed guide on how to setup motive.
```
surely this will work regardless of the number of cameras? practically, sure you'll want two but it dooesn't stop it running.